cmake/JoinPaths.cmake

```diff
@@ -0,0 +1,26 @@
+# This module provides function for joining paths
+# known from from most languages
```

Comment:
`s/from from/from/`

Reply:
This was copy/pasted from https://github.com/jtojnar/cmake-snips/blob/master/CMakeScripts/JoinPaths.cmake
I've addressed the typo.